contracts/src/SuccinctStaking.sol

```diff
@@ -114,15 +114,16 @@ contract SuccinctStaking is
         uint256 _slashCancellationPeriod
     ) external initializer {
         // Ensure that parameters critical for functionality are non-zero.
-        if (
-            _owner == address(0) || _governor == address(0) || _vApp == address(0)
-                || _prove == address(0) || _intermediateProve == address(0) || _dispenser == address(0)
-        ) {
-            revert ZeroAddress();
-        }
-        if (_maxUnstakeRequests == 0 || _unstakePeriod == 0 || _slashCancellationPeriod == 0) {
-            revert ZeroAmount();
-        }
+        _requireNonZeroAddress(_owner);
+        _requireNonZeroAddress(_governor);
+        _requireNonZeroAddress(_vApp);
+        _requireNonZeroAddress(_prove);
+        _requireNonZeroAddress(_intermediateProve);
+        _requireNonZeroAddress(_dispenser);
+        _requireNonZeroNumber(_minStakeAmount);
```

Comment:
this wasn't checked in the old initializer

Reply:
removed